lib/gooddata/rest/phmap.rb

```diff
@@ -9,72 +9,107 @@ module Rest
     class Connection
       # PH_MAP for wildcarding of URLs in reports
       PH_MAP = [
-        ['/gdc/account/profile/{id}', %r{/gdc/account/profile/[\w]+}],
-        ['/gdc/account/login/{id}', %r{/gdc/account/login/[\w]+}],
-        ['/gdc/account/domains/{id}/users?login={login}', %r{/gdc/account/domains/[\w\d-]+/users\?login=[^&$]+}],
-        ['/gdc/account/domains/{id}', %r{/gdc/account/domains/[\w\d-]+}],
+        ['/gdc/account/profile/{id}', %r{/gdc/account/profile/[^\/]+}],
+        ['/gdc/account/login/{id}', %r{/gdc/account/login/[^\/]+}],
+        ['/gdc/account/domains/{id}/users?login={login}', %r{/gdc/account/domains/[^\/]+/users\?login=[^&$]+}],
+        ['/gdc/account/domains/{id}', %r{/gdc/account/domains/[^\/]+}],
 
-        ['/gdc/projects/{id}/execute', %r{/gdc/projects/[\w]+/execute}],
+        ['/gdc/projects/{id}/execute', %r{/gdc/projects/[^\/]+/execute}],
 
-        ['/gdc/datawarehouse/instances/{id}', %r{/gdc/datawarehouse/instances/[\w]+}],
-        ['/gdc/datawarehouse/executions/{id}', %r{/gdc/datawarehouse/executions/[\w]+}],
+        ['/gdc/dataload/projects/{id}/outputStage', %r{/gdc/dataload/projects/[^\/]+/outputStage}],
 
+        ['/gdc/datawarehouse/instances/{id}', %r{/gdc/datawarehouse/instances/[^\/]+}],
+        ['/gdc/datawarehouse/executions/{id}', %r{/gdc/datawarehouse/executions/[^\/]+}],
+
+        # domain dataproducts' clients
+        ['/gdc/domains/{id}/dataproducts/{data_product}/clients?segment={segment}',
+         %r{\/gdc\/domains\/[^\/]+\/dataproducts\/[^\/]+\/clients\?.*segment=[^\/]+}],
+        ['/gdc/domains/{id}/dataproducts/{data_product}/clients/{client}', %r{/gdc/domains/[^\/]+/dataproducts/[^\/]+/clients/[^\/]+}],
+        ['/gdc/domains/{id}/dataproducts/{data_product}/clients', %r{/gdc/domains/[^\/]+/dataproducts/[^\/]+/clients}],
+
+        # domain dataproducts' segments
+        ['/gdc/domains/{id}/dataproducts/{data_product}/segments/{segment}/synchronizeClients/results/{result}/details?offset={offset}&limit={limit}',
+         %r{/gdc/domains/[^\/]+/dataproducts/[^\/]+/segments/[^\/]+/synchronizeClients/results/[^\/]+/details/\?offset=[\d]+&limit=[\d]+}],
+        ['/gdc/domains/{id}/dataproducts/{data_product}/segments/{segment}/synchronizeClients/results/{result}',
+         %r{/gdc/domains/[^\/]+/dataproducts/[^\/]+/segments/[^\/]+/synchronizeClients/results/[^\/]+}],
+        ['/gdc/domains/{id}/dataproducts/{data_product}/segments/{segment}', %r{/gdc/domains/[^\/]+/dataproducts/[^\/]+/segments/[^\/]+}],
+        ['/gdc/domains/{id}/dataproducts/{data_product}/segments', %r{/gdc/domains/[^\/]+/dataproducts/[^\/]+/segments}],
+
+        # domain dataproducts' provision & update
+        ['/gdc/domains/{id}/dataproducts/{data_product}/provisionClientProjects/results/{result}/details?offset={offset}&limit={limit}',
+        %r{/gdc/domains/[^\/]+/dataproducts/[^\/]+/provisionClientProjects/results/[^\/]+/details/\?offset=[\d]+&limit=[\d]+}],
```

Comment:
Style/AlignArray: Align the elements of an array literal if they span more than one line.